packages/vx-bounds/src/enhancers/withBoundingRects.tsx

```diff
@@ -39,15 +42,15 @@ export default function withBoundingRects(BaseComponent) {
     }
 
     componentDidMount() {
-      this.node = ReactDOM.findDOMNode(this);
+      this.node = ReactDOM.findDOMNode(this) as HTMLElement;
```

Comment:
Should we use `ref` instead of doing `ReactDOM.findDOMNode()`?

Reply:
there's been some discussion in the past about this. agree `findDOMNode` is gross 🤮 but it would be a breaking change unless we add an extra div wrapper, and that would also possibly break things for consumers. I think prob best to make that change in another PR.